README.md

```diff
@@ -56,4 +56,11 @@ The [Resource Management](/docs/articles/Resources/ResourceManagement.md) holds
 
 ## Product Management
 
-The [Product Management](/docs/articles/Resources/ResourceManagement.md) holds all product variantes, created instances of those variants and recipes how to create an instance. It provides an API giving access to product types, instances, recipes and workplans.  
+The [Product Management](/docs/articles/Resources/ResourceManagement.md) holds all product variantes, created instances of those variants and recipes how to create an instance. It provides an API giving access to product types, instances, recipes and workplans. 
```

Comment:
Same here, please use UUIDs

Reply:
As you can see above there is already a mixture of refs to md and uuids in almost all md files. But I will switch to uuids in the future. Where should the generated doc be found on github?

Reply:
The generated documentation cannot be found anywhere. This is an open point. I added an issue to the platform issue tracker.

Reply:
In this readme, direct links to files are okay, my fault